python/lsst/afw/display/interface.py

```diff
@@ -29,6 +29,9 @@
 import importlib
 import lsst.afw.geom  as afwGeom
 import lsst.afw.image as afwImage
+import lsst.log
+
+logger = lsst.log.Log.getLogger("afw.display.interact")
```

Comment:
I actually don't know the answer to this: do we have a convention for choosing logger names?

Regardless of a convention, the above looks a bit suspicious because it's in scope for the whole file but calls out `interact`, which is one specific method in one class contained in the file, by name.